### ISSUE TITLE
Feature Request: Test Case description / support Markdown

### ISSUE THREAD

**Issue by bbbco** [leoGalani/sherlock#57]
Right now if you add multiple lines and other formatting to a test case, it gets saved on a single line.
It'd be nice to support a further description on each test case.

In fact, it would be really helpful to support Markdown formatting.

**Comment by teixeira-fernando** [comment]
I agree with @bbbco. It would be a very helpful feature to implement.
